scalafmt-tests/src/test/resources/unit/For.stat

```diff
@@ -23,12 +23,28 @@ row <- table) yield ( for (
 cell <- row) yield if (cell == null) 0
 else cell.toString.length)
 >>>
-val sizes = for (row <- table)
-  yield
-    (for (cell <- row)
+val sizes =
+  for (row <- table)
+    yield (for (cell <- row)
       yield
         if (cell == null) 0
         else cell.toString.length)
+<<< crazy one-liner 2
+val sizes = for (row <- table) yield (
+  for (cell <- row)
+  yield
+    if (cell == null) 0
+    else cell.toString.length
+)
+>>>
+val sizes =
+  for (row <- table)
+    yield (
+        for (cell <- row)
```

Comment:
FTR, the default indent is 2 spaces but for legacy reasons the tests use 4 space continuation indent